### DISCUSSION TITLE
I keep getting "python.app" requesting incoming request connection , even though I have granted the permission

### DISCUSSION THREAD

**Discussion by hari2262** [maxileith/homebridge-appletv-enhanced#207] [Q&A]
I keep getting "python.app" requesting incoming request connection , even though I have granted the permission
OS: macOS Catalina
Node.js Version(Homebridge): v20.11.0
App Version: v1.20

TIA
![Screen Shot 2024-01-26 at 10 55 02 AM](https://github.com/maxileith/homebridge-appletv-enhanced/assets/6504053/81097053-c457-4a81-8cf4-df36ad62ecda)

**Comment by maxileith** [comment]
MacOS is not supported. I don't have experience with MacOS. However, you might get an answer from the community.
